pkg/network/protocols/http/monitor.go

```diff
@@ -79,6 +86,102 @@ func NewMonitor(c *config.Config, offsets []manager.ConstantEditor, sockFD *ebpf
 		return nil, fmt.Errorf("error initializing http ebpf program: %w", err)
 	}
 
+	staticTable, _, _ := mgr.GetMap(string(probes.StaticTableMap))
+	if staticTable != nil {
+		type staticTableEntry struct {
+			Index uint64
+			Value StaticTableValue
+		}
+
+		staticTableEntries := []staticTableEntry{
+			{
+				Index: 2,
+				Value: StaticTableValue{
+					Key:   MethodKey,
+					Value: GetValue,
+				},
+			},
```

Comment:
shouldn't the index be equal to the key?

Reply:
it is not possible due to the fact that the key can be equal between 2 different entries, for example lets take the method key which is equal to 2, it should me the key "method" for the post and get as well, if we will use the key for the index we will not be able to distinguish between them.